packages/gatsby-theme-sso-ui-kit/src/@commercetools-docs/gatsby-theme-docs/overrides/avatar.js

```diff
@@ -208,21 +139,19 @@ const UserProfile = () => {
       });
     }
 
-    return <UserInformation userData={userData} logout={logout} />;
+    return <LoggedInState userData={userData} logout={logout} />;
```

Comment:
I think we are still pushing `association` in the `userData` object. That should be not needed anymore as we don't display the company information anymore